packages/cloud-cognitive/src/components/ImportModal/ImportModal.js

```diff
@@ -144,20 +145,21 @@ export let ImportModal = ({
       primaryButtonDisabled={primaryButtonDisabled}
       onRequestSubmit={onSubmitHandler}
       onRequestClose={onRequestClose}
-      className={`${pkg.prefix}--import-modal`}
+      className={blockClass}
       size="sm">
-      <p className={`${pkg.prefix}--import-modal-body`}>{modalBody}</p>
-      <p className={`${pkg.prefix}--import-modal-label`}>{fileDropHeader}</p>
+      <p className={`${blockClass}-body`}>{modalBody}</p>
```

Comment:
These should be of the form `${blockClass}__body` our BEM format is as per Carbon `prefix--Block__Element--Modifier`